components/nimbus/ios/Nimbus/NimbusApi.swift

```diff
@@ -22,7 +22,7 @@ public protocol NimbusFeatureConfiguration {
     /// - Parameter featureId The string feature id that applies to the feature under experiment.
     /// - Returns A String representing the branch-id or "slug"; or `nil` if not enrolled in this experiment.
     ///
-    func getExperimentBranch(featureId: String) -> String?
+    func getExperimentBranch(experimentId: String) -> String?
```

Comment:
Renamed now that we're moving away from branch names.

This method is still used, but only for tooling.